smt/available_solvers.cpp

```diff
@@ -225,6 +231,9 @@ SmtSolver create_interpolating_solver_for(SolverEnum se, Engine e)
 {
   if (ic3_variants().find(e) == ic3_variants().end()) {
     return create_interpolating_solver(se);
+  } else if (se != MSAT) {
+    throw SmtException(
+        "Running IC3-based engine with interpolation requires MathSAT.");
```

Comment:
Why did you add this? It should also work with cvc5, right?

Reply:
True, I've fixed it in d4aa6b052231212a0e5aadd8f078fdd1bd65236b